src/modules/commands/discord/certs.ts

```diff
@@ -0,0 +1,30 @@
+import discord from 'discord.js'
+import { Command } from '@commands/CommandHandler'
+import { censusApi } from '@app/modules/planetside/CensusApi'
+import { getEmoji } from '@app/modules/discord/utils'
+
+export default new Command<discord.Message>({
+  keyword: 'certs',
+  description: 'shows available certs of any player',
+  help: 'Usage: `{prefix}certs <player name>` - shows available certs of any player',
+  callback: async ({ args, reply, env, raw }) => {
+    if (args.length === 0) {
+      return reply(env.command.getHelp(env.handler))
+    }
+    if (args.length > 1) {
+      return reply('Player names cannot contain any spaces.')
+    }
+    const characterName = args[0].toLowerCase()
+    const character = await censusApi.getCharacterCerts(characterName)
+    if (character === null) {
+      return reply('There is no PlanetSide 2 character with this name.')
+    }
+    reply(
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+      `**${character.name} has ${character.available_certs}${getEmoji(
```

Comment:
In case of missing emoji, replace it with the word "certs".

Reply:
This gets you rid of the // eslint-disable. In the future if you really need // eslint-disable, please comment why it is necessary.